modules/standard/FileSystem.chpl

```diff
@@ -173,6 +173,54 @@ proc exists(name: string): bool {
   return ret;
 }
 
+/* Returns the user id associated with the file or directory specified by
+   name.  Returns an errors that occurred via an out parameter.
+   err: a syserr used to indicate if an error occurred
+   name: a string used to indicate the file in question
+*/
+proc getUID(out err: syserr, name: string): int {
+  extern proc chpl_fs_get_uid(ref result: c_int, filename: c_string): syserr;
+
+  var result: c_int;
+  err = chpl_fs_get_uid(result, name.c_str());
+  return result;
+}
+
+/* Returns the user id associated with the file or directory specified by
+   name.  Returns an errors that occurred via an out parameter.
```

Comment:
This one does not return errors via an out parameter.  Same with the second getGID function.